ipykernel/ipkernel.py

```diff
@@ -71,6 +72,8 @@ class IPythonKernel(KernelBase):
     shell = Instance("IPython.core.interactiveshell.InteractiveShellABC", allow_none=True)
     shell_class = Type(ZMQInteractiveShell)
 
+    debugger_class = Type(Debugger)
```

Comment:
You can also give this class as a full-specified dotted string, so we could move the import above back to being an on-demand import in the class definition.

Reply:
I think this should work:
```suggestion
    debugger_class = Type("ipykernel.debugger.Debugger")
```

Reply:
Thanks for the suggestion; I think if I do this I can omit the import altogether?